org.eclipse.lsp4xml.extensions.maven/src/main/java/org/eclipse/lsp4xml/extensions/maven/MavenCompletionParticipant.java

```diff
@@ -82,27 +89,45 @@ public void onTagOpen(ICompletionRequest request, ICompletionResponse response)
 
 	@Override
 	public void onXMLContent(ICompletionRequest request, ICompletionResponse response) throws Exception {
+		if (request.getXMLDocument().getText().length() < 2) {
+			CompletionItem item = new CompletionItem("minimal pom content");
```

Comment:
Looks good to me, great idea :)
Maybe this should be wrapped into a function, however?